### PR TITLE
Aligned the image to center and removed extra margin bellow 767px

### LINKED ISSUE #3325: Alignment of Image looks odd below 767px on page: /service-mesh-books
#### Description
Alignment of Image looks odd below 767px on page: /service-mesh-books 
Screenshot 2022-10-15 104825

#### Expected Behavior
Screenshot 2022-10-15 104901

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sam1code**
…it look ugly bellow width 767px

Signed-off-by: Soumyakanta Pattanaik <samcse95@gmail.com>

**Description**
Aligned the image to center and removed extra margin which is making it look ugly bellow width 767px.

This PR fixes #3325

Screenshot from 2022-10-20 02-12-49

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e7b3391dfcd75af1e858ff467eb230ee45da2417 at: https://63506495ba4a2708ad283737--layer5.netlify.app

**Comment by sam1code** [comment]
Can anyone check this please! Else it might get old and conflicted!

**Comment by l5io** [comment]
🚀 Preview for commit ebf2bf18ad4a63da1269b2d7aaa149d39f0e8d05 at: https://63530b9b19d73b007ad36ac0--layer5.netlify.app

File: src/sections/Learn/Learn-Service-Mesh-CTA/index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const LearnServiceMeshCTAWrapper = styled.div`
     .section-wrapper{
       @media (max-width: 767px){
           flex-direction: column-reverse;
+          margin-top: -4rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Learn/Learn-Service-Mesh-CTA/index.js:20]:
```suggestion
          margin-top: -6rem;
```

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**